spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/msi/AzureInstanceMetadataService.java

```diff
@@ -15,9 +15,14 @@
 
 /**
  * Wraps functions provided by Azure Instance Metadata Service(IMDS)
+ *
+ * REST endpoint URL and parameters are defined in below document:
+ * <pre>
+ * @see <a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http">Get a token using HTTP<a/>
+ * </pre>
  */
 public class AzureInstanceMetadataService {
-    private String apiVersion = "2018-02-01-preview";
+    private String apiVersion = "2018-02-01";
     private String tokenEndpoint = "http://169.254.169.254/metadata/identity/oauth2/token";
```

Comment:
Why are we using a raw IP address ?

Reply:
Answered my own question: [link](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token)